### ISSUE TITLE
"On Manual Interaction Required" notification not working

### ISSUE THREAD

**Issue by sschiphorst** [Sonarr/Sonarr#8148]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

I have set up a connection to send a notification when a manual interaction is required. I get an notification when there is a 'waiting to import' for 'all season in one file' but not for 'sample file only' which occurs far more often.

### Expected Behavior

Reading [https://github.com/Sonarr/Sonarr/issues/6448](url) I would expect that the 'sample file only' should trigger the same notification to be sent as any other error causing a file to be stuck 'waiting for import'

### Steps To Reproduce

- Sonarr has a download that it can't import automatically.
- In my case, the name of the downloaded file is a sample file only not the actual episode.

### Environment

```markdown
- OS: QTS 5.2.6
- Sonarr: 4.0.15.2941
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser: N/A
- Database: Sqlite 3.48.0
```

### What branch are you running?

Main

### Trace Logs?

https://gist.github.com/sschiphorst/3a371836d8c4d5d90888562dce0dc170

**Comment by markus101** [comment]
They are only triggered in certain situations, a file being a sample isn't one of those currently (and no plans to implement that at this time).

Your logs seem to show an issue getting sample information, so the import process is failing, which isn't covered by this notification.

**Comment by sschiphorst** [comment]
Can I ask why it's not planned to be covered given that it's an item that requires manual interaction to solve which is what that notification is designed to cover? Just trying to understand if I'm fundamentally misunderstanding the root of the issue.
